docs/reference/data-sources/kinesis.md

```diff
@@ -0,0 +1,72 @@
+# Kinesis source
+
+**Warning**: This is an _experimental_ feature. It's intended for early testing and feedback, and could change without warnings in future releases.
+
+## Description
+
+Kinesis sources allow users to register Kinesis streams as a data source. Feast currently does not manage the ingestion of data from Kinesis streams; users must manage the ingestion themselves and write feature values to the online store through [FeatureStore.write_to_online_store](https://rtd.feast.dev/en/latest/index.html#feast.feature_store.FeatureStore.write_to_online_store).
+
+Kinesis sources must have a batch source specified, since that's the source used when retrieving historical features. When using a Kinesis source as a stream source in the definition of a feature view, a batch source doesn't need to be specified in the definition explicitly.
```

Comment:
Doesn't need to be specified in the feature view definition i presume?

Reply:
clarified